### ISSUE TITLE
Voiceover does not automatically shift focus to the "Category" popup when it opens.: A11y_Developer balance version .NET 10_Project_ScreenReader

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#30746]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG1.3.2; #A11ySev2; #iOS; #BM_.NET maui_July2025; #.NET MAUI SDKs; #Device; #ScreenReader; #Voiceover; #AINotInscope; #iPhone; #Meaningful sequence;

### Environment Details:
Application Name: Developer balance version .NET 10
OS: iOS
Model Name: iPhone 15
iOS version: 18.5
Screen reader: Voiceover

### Repro Steps:

1. Start Voiceover.
2. Open "Developer balance" app.
3. "Home Screen" will open.
4. Swipe till any created project and double tap on it to select.
5. Swipe till "category" and double tap on it. Popup will appear.
6. Observe that Voiceover does not automatically shift focus to the "Category" popup when it opens.

### Actual Result:
**Issue 1:** Voiceover does not automatically move focus to the "Category" popup when it opens. When navigating with a right swipe, it goes through all the controls after the 'Category' field before reaching the category popup.

**Issue 2:** When the category popup is closed, the voiceover focus shifts to the tag instead of returning to the category control or the icon control.

### Expected Result:
**Issue 1:** When the 'Category' popup appears, Voiceover should immediately shift focus to the popup and announce its presence, allowing users to interact with it.

**Issue 2:** When the category popup is closed, the voiceover focus should shift to the category control or the icon control.

### User Impact:
Users might not notice when a popup appears, which can cause confusion or result in missed information. If the popup does not receive automatic focus, users have to find and access it themselves, which can be unintuitive and inconsistent, particularly if there is no announcement for the popup.

### Attachment:

Issue 1: 

https://github.com/user-attachments/assets/45f2ced1-bd9b-4523-805c-c528c9b4c812

Issue 2:

https://github.com/user-attachments/assets/1636cde6-df16-4343-a618-2ffadb05d5d6

**Comment by ghost** [comment]
This issue has been verified in Visual Studio Code 1.102.1 with MAUI versions 9.0.0 & 9.0.90. It can be reproduced on iOS platform.

**Comment by PoojaNamde** [comment]
#Regressed:09-23-25; 

Verified the issue in Version: 8.0, issue is not fixed and still reproducing

**Comment by PoojaNamde** [comment]
@PureWeen Since this bug is marked as external to Syncfusion Toolkit Chart, could you please confirm if it is a first-party or third-party issue. If it is first party, please provide the service tree details and product name. If it is third-party, kindly share the sample app so we can log the bugs.

**Comment by Yash14j** [comment]
GitHub Tags:
#External;#ThirdParty;#External:SyncfusionToolkitChart;#ResolvedbyTT;

**Comment by kapilvaishna** [comment]
@PureWeen For issue 1: The focus is now on the value list, but when swiping up or down, it doesn't navigate through the items in the value list. Instead, it exits the value list popup.

Issue 2: When I double tap the "Done" button, the focus shifts to the "Back" button instead of moving to the "Category" combo box.

https://github.com/user-attachments/assets/b3ae6c63-f038-4c76-9815-c1b7fc451689
